src/Utils.js

```diff
@@ -127,7 +127,7 @@ export function getDistance2D(fromx, fromy, tox, toy) /* ,false=false */ {
   w = tox - fromx;
   h = toy - fromy;
 
-  // if (GetDefaultArg(fast, false)) return w * w + h * h;
+  if (GetDefaultArg(fast, false)) return w * w + h * h;
```

Comment:
getdefaultarg doesn't exist anymore, you need to write this without using it

Reply:
The original getDistance2D function has 5 args
function GetDistance2D( fromx, fromy, tox, toy, fast /*= false*/ )

Not sure how to handle this

Reply:
Not sure what I've sone here is good. If an arg is omitted does 'undefined' need to be set as a placeholder?